ext/sockets/sockets.c

```diff
@@ -1952,6 +1953,40 @@ PHP_FUNCTION(socket_set_option)
 		}
 #endif
 
+#ifdef SO_ATTACH_REUSEPORT_CBPF
+		case SO_ATTACH_REUSEPORT_CBPF: {
+			convert_to_long(arg4);
+
+			if (!Z_LVAL_P(arg4)) {
+				ov = 1;
+				optlen = sizeof(ov);
+				opt_ptr = &ov;
+				optname = SO_DETACH_BPF;
+			} else {
+                uint32_t k = (uint32_t)Z_LVAL_P(arg4);
+                static struct sock_filter cbpf[8] = {0};
+                static struct sock_fprog bpfprog;
+
+                switch (k) {
+                case SKF_AD_CPU:
+					cbpf[0].code = (BPF_LD|BPF_W|BPF_ABS);
+                    cbpf[0].k = (uint32_t)(SKF_AD_OFF+k);
```

Comment:
It took me quite a bit of time to realise that this is for getting to current cpu id because I have to realise that k is actually SKF_AD_CPU. I know it is obvious as it is a switch case but it could be a bit clearer just by using `SKF_AD_CPU`

Reply:
also NIT that we usually write a space around `+`...